doc/install-macos.md

```diff
@@ -0,0 +1,83 @@
+# macOS Installation
+
+To install the fluxon-mhd modeling framework, there are three main requirements - a perl installation with libraries, the perl data language toolset, and finally compiling the fluxon-mhd software itself.
+
+Homebrew and the base perl installation will be installed as a user that has full system access, with further packages and flux itself installed locally. This provides both a buffer if the system perl is updated, and also allows regular access for users who do not have regular administrative privileges.
+
+## Perl installation
+
+First, install the homebrew package, which will require being logged in as an admin user. Follow through the installation process.
+```shell
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Next, use homebrew as an admin user to install perl.
+```shell
+brew install perl
+```
+
+If you plan on using homebrew for other packages, and would like to keep perl from updating alongside other packages, the installation can be frozen.
+```shell
+brew pin perl
+```
+
+Two more homebrew packages will be needed later, which can also be installed via an admin user.
+```shell
+brew install gnuplot
+brew install fftw
+```
+
+The final suggested step as an admin user is to setup the cpanminus tool for installing perl packages.
+```shell
+curl -L https://cpanmin.us | perl - --sudo App::cpanminus
+```
+
+Set perl to install packages to a local library directory. Here we've chosen to use a perl5 directory inside the ~/Library folder, but this could be anywhere your user has permissions. This is then stored to automatically update in a .zprofile startup script for Zsh - modify if using a different shell.
+```shell
+PERL_MM_OPT="INSTALL_BASE=~/Library/perl5" cpanm --local-lib=~/Library/perl5 local::lib
+echo 'eval `perl -I ~/Library/perl5/lib/perl5 -Mlocal::lib=~/Library/perl5`' >> ~/.zprofile
+```
+
```

Comment:
potential need for a soource of zprofile here

Reply:
Done.